### PR TITLE
added sketch image converter

### LINKED ISSUE #115: Colored Image to Sketch Converter
## Suggest a new script for the project
Python Image Converter

### Describe the new script you want
I want to add a Python Script which will Convert Colored Image to Sketch Image

### Describe the solution you'd like us to follow
I will user OpenCV library to convert Image to Sketch Image

## Do you want to work on this?
- [x] yes
- [ ] no

### PR DESCRIPTION
**Pull request by deepak-chouhan**
# Description

Please include a general summary of the change and/or which issue it fixed. 

Fixes #115

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] New Script (non-breaking change which adds a new script to the repository)
- [ ] This change requires a documentation update

## Has This Been Tested?

- [x] I confirm that I have tested my script locally and verified that it works as intended.

## Checklist:

- [x] I have gone through the contributing guidelines before making this Pull Request.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by OBITORASU** [comment]
Add your script to main readme please.

**Comment by deepak-chouhan** [comment]
> Add your script to main readme please.

okay

**Comment by deepak-chouhan** [comment]
@OBITORASU I have update the added the script to readme

File: scripts/image_converter/requirements.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Sketch image converter
```
Comment (OBITORASU) [inline scripts/image_converter/requirements.md:1]: This file should be named readme.md
Make a seperate file called requirements.txt for requirements.

Comment (deepak-chouhan) [reply scripts/image_converter/requirements.md]: done

**Comment by deepak-chouhan** [comment]
@OBITORASU , I have added separate requirements.txt file and also renamed the requirements.md to readme.md. please check if there is anything I need to do.

**Comment by OBITORASU** [comment]
Ty merged!

**Pull request merged**